mkt/search/views.py

```diff
@@ -109,18 +115,20 @@ def _sort_search(request, sq, data):
     """
     from mkt.api.base import get_region_from_request
 
+    region = get_region_from_request(request)
+
     # When querying we want to sort by relevance. If no query is provided,
     # i.e. we are only applying filters which don't affect the relevance,
     # we sort by popularity descending.
```

Comment:
Also something you didn't add, but can you add in this comment a "By default" or "If no sort param is provided", something like that ? It's confusing to read considering what happens below when a sort parameter is provided.